roles/ceph-compute/tasks/main.yml

```diff
@@ -40,3 +40,8 @@
 
 - name: set virsh secret
   shell: "virsh secret-set-value --secret {{ cinder_uuid_file.content | b64decode }} --base64 $(cat /etc/ceph/client.cinder.key)"
+  
+- name: copy ceph config for enabling rbd cache
+  synchronize: src=/etc/ceph/ceph.conf dest=/etc/ceph/ceph.conf
```

Comment:
I'm not super thrilled about using the rsync method here to get the content in place. I'm assuming that it's because the template itself requires data from the ceph-common role, but the ceph-common role installs more things than you'd want on a compute node.  That sounds more like we need to break apart ceph-common some more so that it can be a truly common role. So, I'll reluctantly take this in for now, if it works, I'm just not happy about it. Actually, I don't like it at all, because it means the success of ceph-compute depends on ceph_monitor role being ran first, on the proper hosts.